doc/glossary.rst

```diff
@@ -267,18 +267,17 @@
                 meaning is intended. 
 
 	OSD fsid
-                This is a unique identifier used to further improve the
-                uniqueness of an OSD and it is found in the OSD path in a file
-                called ``osd_fsid``. This ``fsid`` term is used interchangeably
-                with ``uuid``
+                This is a unique identifier used to identify an OSD. It is
+                found in the OSD path in a file called ``osd_fsid``. The
+                term ``fsid`` is used interchangeably with ``uuid``
 
 	OSD id
                 The integer that defines an OSD. It is generated by the
-                monitors as part of the creation of a new OSD.
+                monitors during the creation of an OSD.
```

Comment:
suggest s/an/each/

Reply:
You're right. "each" is a lot better in this context.

Approve this PR so that I can raise another PR to ingest this suggestion.

Reply:
@anthonyeleven https://github.com/ceph/ceph/pull/49593